src/components/PrevStats/PreviousStats.jsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Heading,
+  SectionBox,
+  BoxHeader,
+  BoxDetail,
+  BoxContent,
+  BoxText,
+  BoxSubContent,
+  BoxSubHeader,
+  BoxSubDetail,
+  SectionContainer,
+  SectionContent,
+  SectionImage,
+  ProgressNumber,
+  StatNumber,
+  SectionCards,
+  MobileBox1,
+  MobileBoxHeader,
+  MobileBoxContent,
+  MobileContainer,
+  MobileDetail,
+  MobileHead,
+  MobileSubContent,
+  MobileSubBar,
+} from './styles';
+
+import prevstat from '../../../config/content/prevstat';
+
+const PreviousStats = () => {
+  const [windowWidth, setWindowWidth] = useState(() =>
+    typeof window !== 'undefined' ? window.innerWidth : 0,
+  );
+  const [isHovered, setIsHovered] = useState(Array(prevstat.data.length).fill(false));
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowWidth(typeof window !== 'undefined' ? window.innerWidth : 0);
+    };
+
+    if (typeof window !== 'undefined') {
+      window.addEventListener('resize', handleResize);
+
+      return () => {
+        window.removeEventListener('resize', handleResize);
+      };
+    }
+
+    return undefined;
+  }, []); // Empty dependency array ensures that the effect runs once when the component mounts
+
+  const isMobile = windowWidth < 1023;
+
+  const imageUrl =
+    'https://res.cloudinary.com/dzxgf75bh/image/upload/v1701689072/' +
+    'xori1-removebg-preview_1_tbbsw8.png';
+
+  const handleHover = (index) => {
+    setIsHovered((prev) => prev.map((_, i) => i === index));
```

Comment:
This can also be done with CSS. The problem with onHover is it results in janky animation when state changes during animation itself.